frontends/profile/SettingsTab/components/AboutTab/AboutTab.tsx

```diff
@@ -57,6 +57,7 @@ export const AboutTab: FC<INavTab> = ({ currentBlog, res = {} }) => {
       }
       return res;
     }, [] as Array<ILink>);
+    prefixTelegramLink(socialMediaLinks);
```

Comment:
Do not add prefix on saving link, add it on rendering to user. Backend just needs usernames